srcpkgs/ueberzug/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'ueberzug'
+pkgname=ueberzug
+version=18.1.3
+revision=1
+build_style=python3-module
+hostmakedepends="python3-setuptools"
+makedepends="python3-devel libX11-devel libXext-devel"
+depends="libXext python3-xlib python3-pillow-simd python3-docopt python3-psutil python3-attrs"
+short_desc="Command line util which allows drawing images on terminals"
+maintainer="mustaqim <mustaqim@pm.me>"
+license="GPL-3.0-only"
+homepage="https://github.com/seebye/${pkgname}"
+distfiles="${homepage}/archive/${version}.tar.gz"
```

Comment:
Use only `${version}`, otherwise [automatic release checking](https://github.com/void-linux/void-packages/blob/master/Manual.md#checking-for-new-upstream-releases) won't work